htmlmetadata/main.py

```diff
@@ -19,6 +19,8 @@ def extract_metadata(url: str) -> dict:
     if url != response.geturl():
         metadata["request"]["redirects"] = response.geturl
 
+    metadata["request"]["headers"] = dict(response.headers)
```

Comment:
Why the `dict` casting is necessary ?  if it's about copy without doing a reference you could do 

```
     metadata["request"]["headers"] = {**response.headers}
```

Reply:
The headers object is actually a dict-like object. It seemed like a good idea to return a pure dictionary so that it outputs correctly, etc.

I believe `{**response.headers}` is exactly the same as `dict(response.headers)`, like from a bytecode level. Whatever style you prefer is fine.

Reply:
I think you did right, because the json encoding could be an issue for some response.headers values, not sure 100%. The `{ **response.headers }` is fine we will be enforcing more python3 :D

Reply:
If you make the change I will merge! thanks for the work!, please fill free to create and add yourself to a `CONTRIBUTORS` file in the root